app/include/MoonEvent.hpp

```diff
@@ -0,0 +1,21 @@
+//
+// Created by marko on 2/11/25.
+//
+
+#ifndef MOONEVENT_H
+#define MOONEVENT_H
+#include "glm/vec3.hpp"
+
+class MoonEvent {
+public:
+    MoonEvent();
+    glm::vec3 turn_on();
+    glm::vec3 turn_off();
+    int moon_event;
```

Comment:
Gledajući ovu klasu, nije potpuno jasno zašto turn_on i turn_off vraćaju vec3 i šta tačno taj vec3 predstavlja.

Obično, metode turn_on i turn_off sugerišu da se neki proces pokreće ili zaustavlja i da zatim radi u pozadini, bez potrebe da pozivalac dalje upravlja njime. Možda bi bilo korisno razmotriti da li je ovakvo ponašanje optimalno ili bi se moglo jasnije modelovati?